okhttp/src/test/java/io/grpc/okhttp/OkHttpServerTransportTest.java

```diff
@@ -1038,8 +1098,11 @@ private Metadata metadata(String... keysAndValues) {
     return metadata;
   }
 
-  private void shutdownAndVerifyGraceful(int lastStreamId) throws IOException {
-    serverTransport.shutdown();
+  private void maybeShutdownAndVerifyGraceful(int lastStreamId, boolean shutdown)
```

Comment:
Split this into a separate method `verifyGracefulShutdown()` instead of passing a boolean?